packages/quantic/force-app/main/default/lwc/quanticCategoryFacetValue/quanticCategoryFacetValue.html

```diff
@@ -1,25 +1,50 @@
 <template>
-  <li class={categoryFacetLiClass}>  
-    <div class="slds-size_1-of-1 facet__value-container" onclick={onSelect}>
+  <div
+    role="button"
+    aria-pressed={isPressed}
+    class="slds-size_1-of-1 facet__value-container"
+    onclick={onSelect}
+    tabindex="0"
+    onkeydown={onKeyDown}
+    aria-label={ariaLabelVoiceOver}
+  >
+    <template if:true={isPressed}>
+      <span class="nowrap parent-text__bold">{item.value}</span>
+      <span class="facet__number-of-results parent-text__bold slds-var-m-left_xx-small">({item.numberOfResults})</span>
+    </template>
```

Comment:
What is this new "isPressed" ?

Reply:
In the original implementation, the selected value is not rendered using `<c-quantic-category-facet-value>`, unlike other values displayed in the filter. Using `<c-quantic-category-facet-value>` to render selected value, isPressed state needed to passed to <c-quantic-category-facet-value>.

Controlling isPressed within `<c-quantic-category-facet-value>` is also coherent with `<c-quantic-facet-value>`